django/__init__.py

```diff
@@ -21,4 +28,18 @@ def setup(set_prefix=True):
         set_script_prefix(
             "/" if settings.FORCE_SCRIPT_NAME is None else settings.FORCE_SCRIPT_NAME
         )
-    apps.populate(settings.INSTALLED_APPS)
+
+    with ExitStack() as stack:
```

Comment:
I think we want to wrap from `configure_logging` through `apps.populate`, just in case the logging configuration imports parts of the project that run queries.